### ISSUE TITLE
Regression: win_pkg fails in msiexec mode

### ISSUE THREAD

**Issue by highlyunavailable** [saltstack/salt#19796]
```
local:
----------
          ID: Ensure WebDeploy is installed
    Function: pkg.installed
        Name: webdeploy
      Result: False
     Comment: An exception occurred in this state: Traceback (most recent call l
ast):
                File "salt/state.py", line 1529, in call
                File "salt/states/pkg.py", line 831, in installed
                File "salt/modules/win_pkg.py", line 571, in install
              TypeError: extend() takes exactly one argument (2 given)
     Started: 11:45:32.424000
    Duration: 485.0 ms
     Changes:
```

Offending lines: https://github.com/saltstack/salt/blob/v2014.7.1/salt/modules/win_pkg.py#L570-L571

```
           Salt: 2014.7.1
         Python: 2.7.5 (default, May 15 2013, 22:44:16) [MSC v.1500 64 bit (AMD64)]
         Jinja2: 2.7.1
       M2Crypto: 0.21.1
 msgpack-python: 0.4.2
   msgpack-pure: Not Installed
       pycrypto: 2.6
        libnacl: Not Installed
         PyYAML: 3.11
          ioflo: Not Installed
          PyZMQ: 14.1.1
           RAET: Not Installed
            ZMQ: 4.0.4
           Mako: Not Installed
```

The fix is to append an array: `cmd.extend(['msiexec', '/i']) or simply set cmd = ['msiexec', '/i'] similar to how it was being set in 2014.7.0: https://github.com/saltstack/salt/blob/v2014.7.0/salt/modules/win_pkg.py#L569

I'm also unable to just drop a slightly edited module into the _modules directory since I'm using a masterless minion - #14048 is rearing its head in here too.

**Comment by jfindlay** [comment]
@highlyunavailable, thanks for reporting this.

**Comment by jfindlay** [comment]
@highlyunavailable, this should make it into the 2014.7.1 release, awesome!  Thanks for reporting.

**Comment by highlyunavailable** [comment]
Oh, I was under the impression 2014.7.1 was already released since there was a windows download for it! Welp.

I've got 1 more that I'll submit a PR for. This does fix the issue, confirmed by me.

**Comment by jfindlay** [comment]
We've made the windows packages already, I believe, but are waiting for the community members who package various distros to package for their respective distros until it becomes 'official'.  In any case, this fix will definitely be in 2014.1, thanks.
